### PR TITLE
Change copyright of minirepro and gpsd

### PR DESCRIPTION
**Pull request by DevChattopadhyay**
Removing the copyright statement from the output of minirepro/gpsd and
updating the copyright in the comments.

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

**Review by gpopt — APPROVED** [review]
+1

File: gpMgmt/bin/gpsd [hunk 1]
```diff
@@ -186,7 +186,7 @@ def main():
         with closing(connection.cursor()) as cursor:
             num_segments = get_num_segments(cursor)
     sys.stdout.writelines(['\n-- Greenplum database Statistics Dump',
-                           '\n-- Copyright (C) 2007 - 2014 Pivotal'
+                           '\n-- Copyright (C) 2007 - 2023 VMWare'
```
Comment (dgkimura) [inline gpMgmt/bin/gpsd:189]: Would it make sense to generate using the current year (i.e. `timestamp.year`)?

**Review by dgkimura — APPROVED** [review]

**Comment by nimish350** [comment]
@kmacoskey Could you please review and approve this change?

File: gpMgmt/bin/gpsd (hunk 1 above)
Comment (DevChattopadhyay) [reply gpMgmt/bin/gpsd]: Thanks @dgkimura for pointing this out. I have made the required changes. Can you please take a second look and let me know if its fine now.

**Review by kmacoskey — CHANGES_REQUESTED** [review]
Copyright should stay the same. It denotes that changes were made between those dates from Pivotal. That is accurate. 

When we make new changes, then we add a new copyright line for VMware. Those copyright lines should look like this. The exact format and words are important. 

```
Copyright (c) [Year(s)] VMware, Inc. or its affiliates. All Rights Reserved.
```

As a different suggested change, I don't feel we need to include a copyright statement in the output of miniRepro or gpsd.  The changes to the code itself are what we are claiming a copyright on. The output of running that code doesn't need that.

**Comment by nimish350** [comment]
> Copyright should stay the same. It denotes that changes were made between those dates from Pivotal. That is accurate.
> 
> When we make new changes, then we add a new copyright line for VMware. Those copyright lines should look like this. The exact format and words are important.
> 
> ```
> Copyright (c) [Year(s)] VMware, Inc. or its affiliates. All Rights Reserved.
> ```
> 
> As a different suggested change, I don't feel we need to include a copyright statement in the output of miniRepro or gpsd. The changes to the code itself are what we are claiming a copyright on. The output of running that code doesn't need that.


Thank you @kmacoskey. We are going to remove the copyright statement from the output file and include it in the comments in the code itself.

**Comment by DevChattopadhyay** [comment]
> Copyright should stay the same. It denotes that changes were made between those dates from Pivotal. That is accurate.
> 
> When we make new changes, then we add a new copyright line for VMware. Those copyright lines should look like this. The exact format and words are important.
> 
> ```
> Copyright (c) [Year(s)] VMware, Inc. or its affiliates. All Rights Reserved.
> ```
> 
> As a different suggested change, I don't feel we need to include a copyright statement in the output of miniRepro or gpsd. The changes to the code itself are what we are claiming a copyright on. The output of running that code doesn't need that.

Hello @kmacoskey 
I have updated as per your comments. Can you please take a look and approve it if it looks good

**Review by kmacoskey — APPROVED** [review]

**Pull request merged**